samcli/commands/build/command.py

```diff
@@ -136,6 +137,16 @@
     help="Enabled parallel builds. Use this flag to build your AWS SAM template's functions and layers in parallel. "
     "By default the functions and layers are built in sequence",
 )
+@click.option(
+    "--mount-with",
+    "-mw",
+    type=click.Choice(MountMode.values(), case_sensitive=False),
```

Comment:
Please check how this is done in other places, example: https://github.com/aws/aws-sam-cli/blob/develop/samcli/commands/local/cli_common/options.py#L230